views/index.erb

```diff
@@ -87,6 +87,20 @@ This vote is simply an additional signal for the community to look at. It is not
 We encourage all ether token holders to participate.
 </p>
 
+<h2>Hashrate Voting </h2>
+<p>
+If you are pool operator, please add PPYE to your node's extra-data field to signal for #ProgPoW or PPNO to oppose it.
+</p>    
+
+<h3>How to participate in Hashrate Voting If I'm just mining ether on a pool?<h3>
+<p>
+Ask your mining pool to support ProgPoW voting.    
```

Comment:
Could you please reword this to be unbiased? We want to keep this site as unbiased as possible. Thanks.

Reply:
Okay, I see what you meant here -- "support ProgPoW voting" is not the same thing as "vote to support ProgPoW" -- but it could be clearer. Something like, "add support for hashrate voting" maybe?

Reply:
I've reworded it to avoid "support" completely.